packages/ui-bindings/agent/agent.tsx

```diff
@@ -84,6 +84,10 @@ query($token: String, $agentId: Int) {
         image
         note
         category
+        currentLocation {
```

Comment:
Why did you add this?

Reply:
The API changed. After I pulled the latest from api-extensions upstream, some new elements were added